### ISSUE TITLE
[Bug] Location cannot be determined

### ISSUE THREAD

**Issue by collectorgeneral** [redfish64/TinyTravelTracker#39]
The app cannot, for whatever reason, determine my location what so ever. I tried waiting a few minutes, but it just does not seem to lock on. 

In case you need a logcat:
[20160818_TTT_Log.txt](https://github.com/redfish64/TinyTravelTracker/files/425908/20160818_TTT_Log.txt)

**Comment by redfish64** [comment]
Tiny Travel Tracker tries to be mindful of the battery. If it left the GPS on all the time, the battery would drain quickly so it uses a different approach.

It has two systems, one is a quick check that turns on the GPS for a few seconds every 5 minutes.

The second is a progressive check, which will turn on GPS for a short time initially. If it can't get a signal, it will wait, and then try again for a longer period of time. It will continue this process, continuously increasing the amount of time waited between checks and the length of time the GPS is turned on for each check, until it finds a signal. At that time, it will restart the process.

The first system is meant to occasionally check if the user has moved to another area where the GPS works better.

The second system is meant to handle cases where the user is in an environment with poor GPS reception, balancing between collecting points and saving battery.

You may want to increase the "Time GPS is on" value in the settings. However, this will also decrease battery life. If you just want to test it, then temporarily increase this value to 100% and it should be able to get a GPS point after a few minutes (given the area you are in has a good enough GPS signal)

**Comment by collectorgeneral** [comment]
@redfish64 
Yes, I did leave it at 100% and tried to wait for a few minutes. It did not work. 
HOWEVER, there was an update this morning, so I updated it, restarted my phone and tried doing it again and it worked! 
Hmmm.. Maybe it was something else that might have been causing the issue? Could be the case. If that is the case, then my bad, Sorry! I should have been a bit more thorough in my analysis.

Thanks for your response!
